picard/util/tags.py

```diff
@@ -140,22 +259,93 @@ def insert(self, index, tagvar):
     def __repr__(self):
         return f"TagVars({self._items!r})"
 
-    def display_name(self, name):
-        tagdesc = None
+    def item_from_name(self, name):
         if ':' in name:
             name, tagdesc = name.split(':', 1)
+        else:
+            tagdesc = None
+
+        if name and name.startswith('_'):
+            search_name = name.replace('_', '~', 1)
+        elif name and name.startswith('~'):
+            search_name = name
+            name = name.replace('~', '_')
+        else:
+            search_name = name
+
+        item: TagVar = self._name2item.get(search_name, None)
+
+        return name, tagdesc, search_name, item
+
+    def script_name_from_name(self, name):
+        tagname, tagdesc, search_name, item = self.item_from_name(name)
+        if item:
+            return str(item)
+        return
```

Comment:
Other functions were not expected to return anything, but this one returns a string (or None).
Even though None is the default we usually explicitly return None in such functions.
In this specific case, we could even just not call return at all since that's the end of the function, but let's be explicit in this case and keep `return None`. 

```suggestion
        return None
```